tenant/middleware_urm_auth.go

```diff
@@ -21,13 +22,23 @@ func NewAuthedURMService(orgSvc influxdb.OrganizationService, s influxdb.UserRes
 }
 
 func (s *AuthedURMService) FindUserResourceMappings(ctx context.Context, filter influxdb.UserResourceMappingFilter, opt ...influxdb.FindOptions) ([]*influxdb.UserResourceMapping, int, error) {
+	orgID := kithttp.OrgIDFromContext(ctx) // resource's orgID
+
+	if feature.OrgOnlyMemberList().Enabled(ctx) {
+		// Check if user making request has read access to organization prior to listing URMs.
+		if orgID != nil {
+			if _, _, err := authorizer.AuthorizeReadResource(ctx, influxdb.OrgsResourceType, *orgID); err != nil {
```

Comment:
Does this get transformed to a 404-style response on the outbound? Could we verify that. I wouldn't want to start leaking knowledge of resources that are outside of scope of visibility by returning the unauthorized error directly.

Reply:
It was saying "unauthroized" I have made an update to force the "not found" response